pubspec.yaml

```diff
@@ -7,12 +7,17 @@ environment:
   sdk: ^3.1.1
 
 dependencies:
+  code_builder: ^4.7.0
+  dart_style: ^2.3.3
+  geojson: ^1.0.0
+  geopoint: ^1.0.0
+  latlong2: ^0.8.2
+  point_in_polygon: ^1.0.0
+  yaml: ^3.0.2
```

Comment:
It should be possible to keep most of the dependencies as `dev_dependencies` except those imported in `country_coder_super.dart` or the **generated** class file.